addons/payment/models/payment_acquirer.py

```diff
@@ -271,974 +316,192 @@ def _prepare_account_journal_vals(self):
-        if not values.get('billing_partner_address'):
-            values['billing_address'] = _partner_format_address(values.get('billing_partner_street', ''), values.get('billing_partner_street2', ''))
-        if not values.get('billing_partner_country') and values.get('billing_partner_country_id'):
-            values['billing_country'] = self.env['res.country'].browse(values.get('billing_partner_country_id'))
-
-        # compute fees
-        fees_method_name = '%s_compute_fees' % self.provider
-        if hasattr(self, fees_method_name):
-            fees = getattr(self, fees_method_name)(values['amount'], values['currency_id'], values.get('partner_country_id'))
-            values['fees'] = float_round(fees, 2)
-
-        # call <name>_form_generate_values to update the tx dict with acqurier specific values
-        cust_method_name = '%s_form_generate_values' % (self.provider)
-        if hasattr(self, cust_method_name):
-            method = getattr(self, cust_method_name)
-            values = method(values)
-
-        values.update({
-            'tx_url': self._context.get('tx_url', self.get_form_action_url()),
-            'submit_class': self._context.get('submit_class', 'btn btn-link'),
-            'submit_txt': self._context.get('submit_txt'),
-            'acquirer': self,
-            'user': self.env.user,
-            'context': self._context,
-            'type': values.get('type') or 'form',
-        })
-
-        _logger.info('payment.acquirer.render: <%s> values rendered for form payment:\n%s', self.provider, pprint.pformat(values))
-        return self.view_template_id._render(values, engine='ir.qweb')
-
-    def get_s2s_form_xml_id(self):
-        if self.registration_view_template_id:
-            model_data = self.env['ir.model.data'].search([('model', '=', 'ir.ui.view'), ('res_id', '=', self.registration_view_template_id.id)])
-            return ('%s.%s') % (model_data.module, model_data.name)
-        return False
-
-    def s2s_process(self, data):
-        cust_method_name = '%s_s2s_form_process' % (self.provider)
-        if not self.s2s_validate(data):
-            return False
-        if hasattr(self, cust_method_name):
-            # As this method may be called in JSON and overridden in various addons
-            # let us raise interesting errors before having stranges crashes
-            if not data.get('partner_id'):
-                raise ValueError(_('Missing partner reference when trying to create a new payment token'))
-            method = getattr(self, cust_method_name)
-            return method(data)
-        return True
-
-    def s2s_validate(self, data):
-        cust_method_name = '%s_s2s_form_validate' % (self.provider)
-        if hasattr(self, cust_method_name):
-            method = getattr(self, cust_method_name)
-            return method(data)
-        return True
-
-    def button_immediate_install(self):
-        # TDE FIXME: remove that brol
```

Comment:
when he was saying "remove that brol", he was talking about the code, not the comment I guess ;-)